### PR TITLE
Fixes #4612 typo in oVirt name

### PR DESCRIPTION
**Pull request by abonas**

### CODE REVIEW

**Comment by abonas** [comment]
Credit for this solution goes to Ohad Levy, thanks Ohad!

**Comment by domcleal** [comment]
@isratrade could you review please?  We should check that there aren't any other instances too, e.g. on the about page, on compute resource listings.

**Comment by domcleal** [comment]
[test]

File: app/helpers/compute_resources_helper.rb [hunk 1]
```diff
@@ -57,4 +57,11 @@ def list_datacenters compute
   rescue
     []
   end
+
+  def list_providers
+    ComputeResource::PROVIDERS.map do |provider|
+      ["#{ComputeResource::STI_PREFIX}::#{provider}".constantize.new.provider_friendly_name, provider]
```
Comment (domcleal) [inline app/helpers/compute_resources_helper.rb:63]: If we change provider_friendly_name to a class method (`def self.provider_friendly_name`), then we can avoid instantiating classes here and just call `constantize.provider_friendly_name`.

Comment (abonas) [inline app/helpers/compute_resources_helper.rb:63]: Not all provider classes have their own provider_friendly_name method. 2 of them inherit from compute_resource.rb, which impl takes name from provider and read_attribute methods, so it means all those need to be changed to be a class method too, no?
isn't it more complicated? unless adding the provider_friendly_name implementation to 2 providers where it's missing...

Comment (domcleal) [inline app/helpers/compute_resources_helper.rb:63]: Hm yeah, I'd probably have added it to the two providers where it's missing in that case so it's always present (i.e. as part of defining a CR).  Which you do is up to you and @isratrade :)

Comment (isratrade) [inline app/helpers/compute_resources_helper.rb:63]: @abonas, @domcleal, I agree with making `provider_friendly_name` a class method.  If compute_resource.rb has  `def self.provider_friendly_name` defined, this should cover for the STI classes missing it, but I would assume that each STI class should define it's own `self.provider_friendly_name`

**Pull request closed without merging**